content/general/concepts/file-paths/file-paths.md

```diff
@@ -13,23 +13,47 @@ CatalogContent:
   - 'paths/computer-science'
 ---
 
-A **file path** describes the location of a file in a computer's folder structure. They are used to locate files like:
+A **file path** specifies the location of a file in a computer's [file system structure](https://www.codecademy.com/resources/docs/general/file-system-structure). They can be used to locate files and web resources, including:
 
-- Media files such as images.
-- Links to [CSS stylesheets](https://www.codecademy.com/resources/docs/css) for the web page.
-- Files that may link to programs such as [CDNs](https://www.codecademy.com/resources/docs/general/cdn), [frameworks](https://www.codecademy.com/resources/docs/general/cdn), or [Node applications](https://www.codecademy.com/resources/docs/open-source/node-js).
-- Other files such as [JSON](https://www.codecademy.com/resources/docs/general/json) or plain text.
+- Media files, such as images
+- Links to [CSS stylesheets](https://www.codecademy.com/resources/docs/css) for a web page
+- Files that may link to programs, such as [CDNs](https://www.codecademy.com/resources/docs/general/cdn), [frameworks](https://www.codecademy.com/resources/docs/general/cdn), or [Node applications](https://www.codecademy.com/resources/docs/open-source/node-js)
+- Other files, such as [JSON](https://www.codecademy.com/resources/docs/general/json) or plain text
+
+## Paths and Path Separators
+
+In general, a path is a string of characters which specifies a unique location in a directory or page hierarchy. For file systems, each level in the hierarchy is a directory. For [URLs](https://www.codecademy.com/resources/docs/general/url), each level in the hierarchy is a page. Different sections of the path are separated by a path separator, such as a forward slash (/). These different sections represent the separate directories or pages in the hierarchy. For example, consider the following file path:
```

Comment:
```suggestion
In general, a path is a string of characters which specifies a unique location in a directory or page hierarchy. For file systems, each level in the hierarchy is a directory. For [URLs](https://www.codecademy.com/resources/docs/general/url), each level in the hierarchy is a page. Different sections of the path are separated by a path separator, such as a forward slash (/). These different sections represent the separate directories or pages in the hierarchy, as shown below:
```

For Docs entries, we'd like the information to be as manual and non-instructive as possible (i.e., "as shown below" instead of "consider the following ...").